nachomemes/store.py

```diff
@@ -149,20 +150,41 @@ def render(self, message: Iterable[str], output: IO):
     }
 )
 
+serializers = {
+    Request: attrgetter("full_url"),
+    float: lambda f: Decimal(str(f)),
+    Font: attrgetter("name"),
+    Color: attrgetter("name"),
+    Justify: attrgetter("name"),
+}
+
+def update_serialization(value: Any, serializers: Dict[Type, Callable]=serializers):
+    if type(value) in serializers:
+        return serializers[type(value)](value)
+    if dict == type(value):
+        return {k: update_serialization(v, serializers) for k, v in value.items()}
+    if type(value) in (list, GeneratorType):
+        return [update_serialization(v, serializers) for v in value]
+    return value
 
 class Store(ABC):
     @abstractmethod
     def refresh_memes(self, guild: Optional[Guild], hard: bool = False) -> str:
         pass
 
     @abstractmethod
-    def read_meme(
+    def get_meme(
         self, guild: Optional[Guild], id: str, increment_use: bool = False
-    ) -> MemeTemplate:
+    ) -> dict:
         pass
 
+    def meme(
```

Comment:
What's the difference between `meme` and `get_meme`?

Reply:
I pointlessly renamed it

Reply:
But `meme` and `get_meme` both exist?

Also, should this be `@abstractmethod`?